showcase/app/templates/layouts/grid.hbs

```diff
@@ -0,0 +1,355 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Layout::Grid Component"}}
+
+<Shw::Text::H1>Layout::Grid</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Column min width</Shw::Text::H2>
+
+  <Shw::Text::H3>gap size 0 (default)</Shw::Text::H3>
+
+  <Shw::Grid @columns="1" @gap="1.5rem" as |SG|>
+    <SG.Item @label="No min width columns (defaults to 0)">
+      <Shw::Outliner>
+        <Hds::Layout::Grid>
+          <Shw::Placeholder @text="#1" @height="24" @background="#e4c5f3" />
+          <Shw::Placeholder @text="#2" @height="24" @background="#e5ffd2" />
+          <Shw::Placeholder @text="#3" @height="24" @background="#d2f4ff" />
+          <Shw::Placeholder @text="#4" @height="24" @background="#fff8d2" />
```

Comment:
[note] in most of my examples, I've changed these placeholders height to `40px` to make it more easy to spot the different blocks; I can do it for you if you want (it's time consuming/boring)